cloud-config/cloud-config-main.yml

```diff
@@ -148,6 +169,10 @@ disk_types:
   cloud_properties:
     type: gp2
     encrypted: true
+- name: kubernetes_storage_class
```

Comment:
See https://github.com/18F/cg-deploy-kubernetes/pull/71/files#r120035804. Kubernetes dynamic volumes are managed by kubernetes and not bosh, so we don't need this.